pmlc/conversion/pxa_to_affine/pxa_to_affine.cc

```diff
@@ -178,8 +189,8 @@ struct AffineReduceOpConversion
   LogicalResult
   matchAndRewrite(pxa::AffineReduceOp op, ArrayRef<Value> operands,
                   ConversionPatternRewriter &rewriter) const final {
-    auto source = rewriter.create<AffineLoadOp>(op.getLoc(), op.mem(), op.map(),
-                                                op.idxs());
+    auto source = rewriter.create<pxa::AffineLoadOp>(op.getLoc(), op.mem(),
```

Comment:
It's not clear to me whether this should be a pxa.load or an affine.load.

Reply:
In PXA -> Affine we want to move back out of PXA and into affine, so the one we are creating wants to be mlir::AffineLoad.

Reply:
The code above works, for what it's worth.  I believe the pxa.load created from the reduce op is later lowered to affine.load.  That said, I will change to the mlir:: version.

Reply:
Yeah, the greedy rewritter keeps rewritting until there are only legal op, but no need to do two rewrites when one will work.